OWNERS_ALIASES

```diff
@@ -6,59 +6,6 @@ aliases:
   - mattmoor
   - tcnghia
 
-  # These aliases are for OWNERS of the various Source implementations. These
-  # Are in addition to the repo level OWNERS.
-
-  awssqs-approvers:
-    - lberk
-  awssqs-reviewers:
-    - lberk
-
-  camel-approvers:
-    - nicolaferraro
-    - matzew
-  camel-reviewers:
-    - nicolaferraro
-    - matzew
-
-  github-approvers:
```

Comment:
Is removing everybody but productivity & toc the owner alias that we really want?

Reply:
I think so since there is a top-level OWNERS file.

Reply:
Ah sorry -- thought we were also flattening the structure as suggested in the camel pr

Reply:
it is flattened.

Reply:
mea culpa!